asr-automation-recovery/azuredeploy.json

```diff
@@ -10,18 +10,7 @@
         },
         "automationRegion": {
             "type": "string",
-            "allowedValues": [
-                "westeurope",
-                "southeastasia",
-                "eastus2",
-                "southcentralus",
-                "japaneast",
-                "northeurope",
-                "canadacentral",
-                "australiasoutheast",
-                "centralindia",
-                "westcentralus"
-            ],
+            "defaultValue": "[resourceGroup().location]",
```

Comment:
this is not a global resource so you need to add allowedValues and remove the defaultValue

Reply:
Reverted the change by removing default value and setting allowed values.